driver/pace/driver/grid.py

```diff
@@ -196,6 +199,110 @@ def get_grid(
         return damping_coefficients, driver_grid_data, grid_data
 
 
+@GridInitializerSelector.register("external")
+@dataclasses.dataclass
+class ExternalGridConfig(GridInitializer):
```

Comment:
If we are bound to the `FRE-NCtools` format we should probably specify that in the name. Looking at the code it looks like a Netcdf4 format, is there a particularity of the output of FRE-NCtools?

Reply:
AFAIK, it's mainly the names of the variables - everything else should be self describing.

Reply:
Let's make it a `ExternalNetcdfGridConfig` or something of the like then, so we are verbose on our expectation